jwst/wfss_contam/disperse.py

```diff
@@ -113,8 +103,8 @@ def flux(x):
     # then finally convert to x/y in the grism image frame
     x0_sky, y0_sky = seg_wcs(x0, y0)
     x0_xy, y0_xy, _, _ = sky_to_imgxy(x0_sky, y0_sky, 1, order)
-    xwmin, ywmin = imgxy_to_grismxy(x0_xy + offset_x, y0_xy + offset_y, wmin, order)
-    xwmax, ywmax = imgxy_to_grismxy(x0_xy + offset_x, y0_xy + offset_y, wmax, order)
+    xwmin, ywmin = imgxy_to_grismxy(x0_xy + xoffset, y0_xy + yoffset, wmin, order)
```

Comment:
I don't see where xoffset/yoffset are being set/initialized before they get used here.

Reply:
Oh wait, they're passed in as arguments in the call to this function, right?

Reply:
Yes, prior to this PR they were unused inputs to this function.